src/QueryM/Ogmios.purs

```diff
@@ -152,59 +151,59 @@ import Untagged.Union (type (|+|), toEither1)
 --------------------------------------------------------------------------------
 
 -- | Queries Ogmios for the system start Datetime
-querySystemStartCall :: JsonWspCall Unit SystemStart
-querySystemStartCall = mkOgmiosCallType Proxy
+querySystemStartCall :: JsonWspCall Unit
```

Comment:
Removing the type parameter leads to less meaningful/informative type signatures for the different calls